gsoc/suborg_guidelines.md

```diff
@@ -133,38 +136,47 @@ has, you will ask for a number of slots to the Open Astronomy administrators.
 They will collect these numbers from all the other sub-org and request them
 to Google.
 
-Google will then, after a few days, tell us how many slots we get. Nobody 
+Google will then, after a few days, tell us how many slots we get. Nobody
 knows them *a priori*, and they vary considerably year after year.
 
-If the number of slots obtained is smaller than the requested then the steering
+If the number of slots obtained is smaller than the requested, then the steering
 council will decide the distribution of the slots using the following points as guidelines:
 
-- The quality of the  student proposal. A well-structured and described project will be favored.
+- The quality of the  student proposal. A well-structured and described project will be favored,
 not simply a copy/paste from the ideas page.
 - What's the likelihood of that candidate student to become a long-term
-contributor? How has the student engaged till now? 
-- Commitment of the mentors for the project. The mentors need to show they have committed to the sub-org and OpenAstronomy, and evidence of this will make the slot more likely to be allocated.
-For example, it is best to have mentors that have contributed to the organisation's code base and are familiar with
-the language and topic.
-- A project that enables cooperation with other sub-organisations of OpenAstronomy will generally be favored over those that do not.
+contributor? How has the student engaged till now?
+- Commitment of the mentors for the project. The mentors need to show they have
+committed to the sub-org and OpenAstronomy, and evidence of this will make the
+slot more likely to be allocated.
+For example, it is best to have mentors that have contributed to the organisation's
+code base and are familiar with the language and topic.
+- A project that enables cooperation with other sub-organisations of OpenAstronomy
+will generally be favored over those that do not.
 
 The process will be as open as possible including only the admins and mentors
-involved in the selection. Remember, we cannot disclose any information on student 
+involved in the selection. Remember, we cannot disclose any information on student
 selection to the students before Google announces the selected students
 
+Note that OpenAstronomy usually assigns one slot for first year sub-orgs.
+That may change in a case-by-case basis, e.g., the sub-organisation already has
+experience with similar coding outreach programmes, such as [ESA-SOCIS][ESA-SOCIS].
```

Comment:
or the suborg participated on their own or under another umbrella in gsoc before. in fact the socis participation is not as strong a prior in my mind (given it's much more relaxed in all sense) than a privious gsoc one.

Reply:
done :)

Reply:
well... relaxed from which point of view? as an org-admin is more stressful than GSOC. It feels more like a :roller_coaster:

Reply:
OK, I see the point. For me from the outside it seems less rigorous, that is rightfully adds stress, but e.g. doesn't test that the sub-org is actually keeps the deadlines, regular meetings with students, not selecting their own students, etc. So SOCIS experience is better than nothing, but still doesn't act as proof for me to see that they can handle multiple students well as a sub-org.